### PR TITLE
fix(db-tool): Tool to run DB migrations

### PR DESCRIPTION
**Pull request by nikurt**

### CODE REVIEW

File: tools/database/src/run_migrations.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+use std::path::Path;
+
+/// This can potentially support db specified not in config, but in command line.
+/// `ChangeRelative { path: Path, archive: bool }`
+/// But it is a pain to implement, because of all the current storage possibilities.
+/// So, I'll leave it as a TODO(posvyatokum): implement relative path DbSelector.
```
Comment (jakmeier) [inline tools/database/src/run_migrations.rs:6]: cc @posvyatokum 
Your team mate is sneaking in a TODO for you here ;)

File: tools/database/src/run_migrations.rs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#[derive(clap::Args)]
+pub(crate) struct RunMigrationsCommand { }
+
+impl RunMigrationsCommand {
+    pub(crate) fn run(&self, home_dir: &Path) -> anyhow::Result<()> {
+        let mut near_config = nearcore::config::load_config(
+            &home_dir,
+            near_chain_configs::GenesisValidationMode::UnsafeFast,
+        )
+            .unwrap_or_else(|e| panic!("Error loading config: {:#}", e));
```
Comment (jakmeier) [inline tools/database/src/run_migrations.rs:23]: needs `cargo fmt`

**Review by jakmeier — APPROVED** [review]

File: tools/database/src/run_migrations.rs (hunk 1 above)
Comment (nikurt) [reply tools/database/src/run_migrations.rs]: That was bad copy-paste. Fixed.

File: tools/database/src/run_migrations.rs (hunk 2 above)
Comment (nikurt) [reply tools/database/src/run_migrations.rs:12]: Done

**Pull request merged**